tasks/deploy_apps.yml

```diff
@@ -0,0 +1,17 @@
+---
+# tasks for managing apps
+# Args:
+#   deployment_state:
+#     - present : default value, all objects are created
+#     - 'absent': all objects are deleted
+#   deployment_stamp : the stamp of the object we going to creat or delete
```

Comment:
What about the following?

```yaml
# Task set to manage apps
# 
# Args:
#   deployment_state:
#     - "present": default value, all objects are created
#     - "absent":  all objects are deleted
#   deployment_stamp: the stamp of the object we are going to create or delete
```

FTR: there is no space before a colon (`:`) in English :wink:

Reply:
ok